ruby/ql/lib/codeql/ruby/frameworks/Railties.qll

```diff
@@ -43,7 +43,7 @@ module Railties {
 
     override DataFlow::Node getAnArgument() { result = this.getArgument([0, 1]) }
 
-    override predicate isShellInterpreted(DataFlow::Node arg) { any() }
+    override predicate isShellInterpreted(DataFlow::Node arg) { arg = this.getAnArgument() }
```

Comment:
Could we restrict this to argument 1?

Reply:
A quick look at the implementation (click 'show source' on [this page](https://apidock.com/rails/v5.0.0.1/Rails/Generators/Actions/execute_command)) suggests that argument 0 would also be shell-interpreted (it gets passed through the `extify` method, but that just adds the `.bat` extension on Windows, and doesn't do any sanitization).